mypy_django_plugin/transformers/models.py

```diff
@@ -747,7 +747,7 @@ def set_auth_user_model_boolean_fields(ctx: AttributeContext, django_context: Dj
 def handle_annotated_type(ctx: AnalyzeTypeContext, django_context: DjangoContext) -> MypyType:
     args = ctx.type.args
     type_arg = ctx.api.analyze_type(args[0])
-    api = cast(SemanticAnalyzer, ctx.api.api)  # type: ignore
```

Comment:
Could we use `helpers.get_semanal_api` instead?

https://github.com/typeddjango/django-stubs/blob/5ba0b063738361c683b1577b5deeec9dee18a448/mypy_django_plugin/lib/helpers.py#L344-L347

Reply:
Yeah, looks about right